### DISCUSSION TITLE
After install extension, it does not show up in dev tools

### DISCUSSION THREAD

**Discussion by jeffschulzusc** [GoogleChromeLabs/ps-analysis-tool#391] [Support Forum] [answered]
**Describe the bug**
I installed the PSAT extension in chrome, verified that is installed and enabled ... but it does not show up in extensions list when i open dev tools. (not under >> nor under more-tools)

**To Reproduce**
Install the PSAT extension in google chrome.
https://chromewebstore.google.com/detail/privacy-sandbox-analysis/ehbnpceebmgpanbbfckhoefhdibijkef

**Expected behavior**
'Privacy Sandbox' should be available in dev-tools.

**Desktop (please complete the following information):**
 - macOS 13.5.2
 - Chrome Browser
 - 120.0.6099.109

**Additional context**
To note, i often have this problem with other extensions, either not showing up or being uninstalled when i update the browser ...
for example, redux tools never shows in devtools panel ... please fix the underlying issue.

**Comment by jeffschulzusc** [comment]
strange, shows up for some site ... but does not show up for my angular SPA site ?

**Comment by gagan0123** [comment, marked as answer]
@jeffschulzusc 

Since you have already installed the PSAT Extension, when trying to audit a site, can you please ensure you take the following steps:

1. Open a new tab with https://example.com/. (Note: This step is crucial due to Chrome's technical and security constraints over extensions. We use example.com specifically because it doesn't add cookies that could interfere with our testing.)
1. Launch Chrome DevTools in the same tab.
1. Within DevTools, locate and click on the "Privacy Sandbox" tab.
1. Navigate to the "Cookies" section under the "Privacy Sandbox" tab.
1. If necessary for the current tab, select "Analyze this tab".
1.  Finally, open the website you wish to audit or analyze in the same tab.

Remember, due to limitations in Chrome, adding a new panel in DevTools via an extension requires an already opened page. Hence, the need for the initial step with example.com.

Here's a screencast for doing the same:
[![PSAT Extension Usage](https://img.youtube.com/vi/7inAXRuu144/maxresdefault.jpg)](https://www.youtube.com/watch?v=7inAXRuu144)

You are welcome to reach out to us for further clarification or assistance.

**Comment by jeffschulzusc** [comment]
ok ... it worked ... thanks much :)

**Comment by maitreyie-chavan** [comment]
Glad the steps worked for you @jeffschulzusc 🙂 

Will go-ahead and close this issue now. In case of any Qs, please do start a discussion in our [Support Forum](https://github.com/GoogleChromeLabs/ps-analysis-tool/discussions/categories/support-forum).
